### ISSUE TITLE
google drive nomenclature

### ISSUE THREAD

**Issue by belgiumkansas** [CU-Robosub/cusub_perception#15]

**Comment by DrYerzinia** [comment]
Is this just deciding what the nomenclature of the files stored in google drive is?  Or documenting it?

I believe @flamma7 has this worked out.

**Comment by flamma7** [comment]
Right, I've landed on an effective nomenclature, I have to rename a few things on the google drive but there already exists documentation
